compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BType.java

```diff
@@ -25,15 +25,7 @@
 import org.wso2.ballerinalang.compiler.util.Names;
 import org.wso2.ballerinalang.compiler.util.TypeTags;
 
-import static org.wso2.ballerinalang.compiler.util.TypeTags.BOOLEAN;
-import static org.wso2.ballerinalang.compiler.util.TypeTags.BYTE;
-import static org.wso2.ballerinalang.compiler.util.TypeTags.DECIMAL;
-import static org.wso2.ballerinalang.compiler.util.TypeTags.ERROR;
-import static org.wso2.ballerinalang.compiler.util.TypeTags.FLOAT;
-import static org.wso2.ballerinalang.compiler.util.TypeTags.INT;
-import static org.wso2.ballerinalang.compiler.util.TypeTags.NIL;
-import static org.wso2.ballerinalang.compiler.util.TypeTags.STRING;
-import static org.wso2.ballerinalang.compiler.util.TypeTags.TYPEDESC;
+import static org.wso2.ballerinalang.compiler.util.TypeTags.*;
```

Comment:
We dont import * right? I think this should be caught by checkstyle

Reply:
i'll change it